src/Razor/test/Microsoft.CodeAnalysis.Razor.Workspaces.Test/UriExtensionsTest.cs

```diff
@@ -93,4 +93,17 @@ public void GetAbsoluteOrUNCPath_UNCPath_HandlesSpacePaths()
         // Assert
         Assert.Equal(@"\\some\path\to\file path.cshtml", path);
     }
+
+    [Fact, WorkItem("https://github.com/dotnet/razor/issues/9365")]
+    public void GetAbsoluteAndUNCPath_HasGitScheme_ReturnsANormalizedFakeGitFilePath()
+    {
+        // Arrange
+        var uri = new Uri("git:/c%3A/myFolder/Index.cshtml?%7B%22path%22%3A%22c%3A%5C%5CmyFolder%5C%5CIndex.cshtml%22%2C%22ref%22%3A%22~%22%7D");
```

Comment:
Would be good to get tests for paths in linux and mac format. If you look at https://github.com/dotnet/razor/blob/main/src/Razor/test/Microsoft.CodeAnalysis.Razor.Workspaces.Test/ProjectSystem/ProjectKeyTests.cs there are some examples of the sorts of things we see in the wild. If you want me to get a real path from viewing a git diff on a mac, let me know. I think Andrew might have a linux machine he can access to try the same thing there.

Reply:
thanks I got a real path from viewing a git diff on mac.

I'll check with @ryzngard for linux too.